saleor/dashboard/discount/forms.py

```diff
@@ -123,94 +139,79 @@ def country_choices():
 
 
 class ShippingVoucherForm(forms.ModelForm):
-
-    limit = MoneyField(
-        min_value=ZERO_MONEY, required=False,
-        currency=settings.DEFAULT_CURRENCY,
-        label=pgettext_lazy(
-            'Lowest value for order to be able to use the voucher',
-            'Only if order is over or equal to'))
-    apply_to = forms.ChoiceField(
+    min_amount_spent = MinAmountSpent
+    countries = forms.MultipleChoiceField(
```

Comment:
Do we really want to limit the choice on the shipping methods?

If I have a country "Rest of the World" in my shipping methods and I want to add a Sale for France for example, I'm not able as France is *not* in the shipping method as France itself.

I think we probably just could retake safely all the country choices without looking over the shipping methods.

https://github.com/mirumee/saleor/blob/a0b69706d6dc6942e252ef9b4977d6121f34b458/saleor/account/i18n.py#L259-L262

Reply:
You can pick 'Rest of the world' inside of the discount as well, I think at some point you might want to offer a free shipping only to specific countries, as shipping abroad might be quite costly

Reply:
Yes, we can pick "Rest of the World" but not "France" if as you say, we want to make discount for that country.

Reply:
I think the root of this dispute is the misleading name of `Rest of the world` country choice, in the codebase, we're calling it `Any Country` which, I think, it's way more suitable for it purpose.

Will change the country choice name, and display countries accordingly